taxonomy/models.py

```diff
@@ -179,7 +178,6 @@ class ProgramSkill(TimeStampedModel):
         )
     )
     confidence = models.FloatField(
-        blank=False,
```

Comment:
why is this removed?

Reply:
pylint says it can be removed as `blant=False` is the default value.